source/particle/interpolator/bilinear_least_squares.cc

```diff
@@ -251,13 +251,15 @@ namespace aspect
 
                     const Postprocess::Particles<dim> &particle_postprocessor =
                       this->get_postprocess_manager().template get_matching_postprocessor<const Postprocess::Particles<dim> >();
-                    const unsigned int n_property_components = particle_postprocessor.get_particle_world().get_property_manager().get_n_property_components();
+                    const auto &particle_property_information = particle_postprocessor.get_particle_world().get_property_manager().get_data_info();
+                    const unsigned int n_property_components = particle_property_information.n_components();
+                    const unsigned int n_internal_components = particle_property_information.get_components_by_field_name("integrator properties");
 
-                    AssertThrow(global_minimum_particle_properties.size() == n_property_components,
+                    AssertThrow(global_minimum_particle_properties.size() == n_property_components - n_internal_components,
```

Comment:
Can you explain why this is necessary? I don't see why the interpolator should have to care how many internal properties there should be...

Reply:
This is still open.

Reply:
I had added a comment before the assert explaining it. Let me know in case the comment is not clear enough.